chamd/chat_reader.py

```diff
@@ -641,17 +641,20 @@ def process_line_steps(linetoprocess: str):
                 if type(step) is ChatLine:
                     if current_line != None:
                         known_line = cast(ChatLine, current_line)
-                        if current_tier != None:
-                            known_tier = cast(ChatTier, current_tier)
-                            known_line.tiers[known_tier.id] = known_tier
-                            current_tier = None
+                        if current_tiers != []:
+                            known_tiers = cast(List[ChatTier], current_tiers)
+                            for kt in known_tiers:
+                                known_line.tiers[kt.id] = kt
+                                current_tiers = []
+
                         chat_file.lines.append(known_line)
                     current_line = step
                     uttid = step.uttid + 1
                 elif type(step) is AppendLine:
-                    if current_tier != None:
-                        cast(ChatTier, current_tier).text += '\n' + \
-                            step.text.lstrip()
+                    if current_tiers != []:
```

Comment:
I'm not 100% on this, what case does this cover?

Reply:
If the previous line didn't contain any tiers.